src-electron/generator/helper-endpointconfig.js

```diff
@@ -93,6 +93,20 @@ function endpoint_fixed_profile_id_array(options) {
   return '{ ' + profileIds.join(', ') + ' }'
 }
 
+/**
+ * Creates array of parent endpoint identifier fields on endpoints
+ *
+ *
+ * @returns C array including the { } brackets
+ */
+function endpoint_fixed_parent_id_array() {
+  return (
+    '{ ' +
+    this.endpoints.map((ep) => ep.parentEndpointIdentifier).join(', ') +
```

Comment:
But not all endpoints have a parent....  Are we using a placeholder value of some sort for "has no parent"?  Something else?

I don't see any tests here exercising this helper.

Reply:
This needs work. 

The idea is the parent value is NULL if there is no parent. 

Working on adding some tests...thanks